packages/composites/market-status/src/config/index.ts

```diff
@@ -0,0 +1,20 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig({
+  TRADINGHOURS_ADAPTER_URL: {
+    description: 'URL of the TradingHours adapter',
+    type: 'string',
+    required: true,
+  },
+  NCFX_ADAPTER_URL: {
+    description: 'URL of the NCFX adapter',
+    type: 'string',
+    required: true,
+  },
+  BACKGROUND_EXECUTE_MS: {
+    description:
+      'The amount of time the background execute should sleep before performing the next request',
+    type: 'number',
+    default: 1_000,
```

Comment:
I think we usually do 10s here, why 1s?

Reply:
This setting effectively determines how stale the data can be right? 10s feels like a long time in that context. Is there an issue with using 1s? That's also how frequently NCFX sends us a WS message for example.

Reply:
Yes, the lower the number -> more request are being sent to APi providers. If they are fine with load we are totally cool